query/src/test/scala/hmda/query/repository/filing/FilingComponentSpec.scala

```diff
@@ -125,6 +123,24 @@ class FilingComponentSpec extends AsyncWordSpec with MustMatchers with FilingCom
 
     }
 
+    "Stream IRS" in {
```

Comment:
Could we get another test that shows that the table is still correct when multiple LAR files get uploaded? (e.g. by multiple institutions or another from the same institution)